src/Extensions/ScriptManager.cs

```diff
@@ -14,6 +14,7 @@ namespace System.Web.UI;
 public class ScriptManager : Control
 {
 
+    [Diagnostics.CodeAnalysis.SuppressMessage("Performance", "CA1810:Initialize reference type static fields inline", Justification = "Initialize something in a different. Should move to DI")]
```

Comment:
Can you make TODO or comment with issue (so that we can clean up all future) ?

Reply:
done